playbooks/deployment-report/library/get_sas_host_details.py

```diff
@@ -1426,7 +1426,7 @@ def _get_process_memory_info(pid, module):
 
     for row in processes:
         # skip blank lines
-        if row is '' or not row:
+        if row == '' or not row:
```

Comment:
this can be simplified to just "if not row", since "not" checks it is both not None and len > 0
```suggestion
        if not row:
```